docs/source/pillars/information_governance.md

```diff
@@ -86,6 +86,14 @@ _The ability of the TRE organisation to monitor compliance with internal and ext
 * - You report on and share outcomes of each audit of your TRE organisation with the required bodies
   - This may included regulatory bodies or the organisations that manage accreditations you have
   - Mandatory
+* - You log user actions, network connection attempts, infrastructure actions and data access within a TRE
+  - Logging actions can help to identify bad behaviours, intentional or unintentional, to be spotted by reviewers.
+    Bad behaviours in this context are anything that could compromise the security of the SRE, or result in the accidental disclosure of data outside the TRE.
+    Any measures that TRE developers deem valuable for reporting should be logged.
+  - Recommended
+* - You make logs public
+  - Where possible, reporting logs should be made publicly available, to incentivise good governance and operation, and should be automated, to enable regular compliance checks.
+  - Optional
```

Comment:
```suggestion
```
I think the only logs we should suggest are made public are data access logs, which might be better in the PIE section?
https://github.com/sa-tre/satre-specification/pull/143
Recording logs comes under IG, but publishing them is separate.

Reply:
I'm not sure what a "reporting log" is. Maybe that is important here?

Publishing logs to prove "we have nothing to hide" feels virtuous. I'm not sure how much use logs are to the public though or what volume we are talking about. Do we believe members of the public are going to be enabled to, or have the enthusiasm to analyse our logs?

Automating the release of logs sounds scary. Have we thought about the risk of logs disclosing information we shouldn't?